### DISCUSSION TITLE
Filtering People search by Office location

### DISCUSSION THREAD

**Discussion by AndyTuke** [microsoft-search/pnp-modern-search#3515] [Q&A] [answered]
If you populate the office property for users in M365, it goes into the Entra property OfficeNumber

![image](https://github.com/microsoft-search/pnp-modern-search/assets/30471813/a9cc6642-4ffb-4042-8ec9-d76d0a39b9b1)

I have added this to the Selected Properties on page 1 of the Search Results web part at which point it appeared in the Filter web part as an option for Edit Filters

![image](https://github.com/microsoft-search/pnp-modern-search/assets/30471813/da2a1705-ecef-44c6-9c25-e86405a68d11)

It just doesnt seem to find those properties to give a filter for them. Is there another place I need to set something?

**Comment by kasperbolarsen** [comment]
Are you referring to office number, like building 4, suite 234, or are you referring to baseofficelocation, the one using the term store values?

**Comment by AndyTuke** [comment]
If I set this for a user in Office 365 admin portal

![image](https://github.com/microsoft-search/pnp-modern-search/assets/30471813/4956650a-8bae-4058-ae1a-2787ece7df85)

It appears in a property returned by the search query tool as shown in my screen shot under Office Number, I dont sync these users in my customer tenant so I cant see how to set BaseOfficeLocation. I assume that is populated from an AD attribute by Entra Connect

**Comment by kasperbolarsen** [comment]
People search in SharePoint Search is based on User Profile properties,  accessible from the SharePoint Admin center. 
In there you have both Office number and Office Location, and can manually update both

**Comment by AndyTuke** [comment]
It appears **Office** syncs to AD, waiting for screenshots from my customer to see what attributes are populated on premise, Does **Office Location**?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/30471813/1550bd62-135c-44bd-8ed1-5161af08fede)

**Reply by AndyTuke** [reply, marked as answer]
For my demo tenant, I've checked the Search configuration and can see that BaseOfficeLocation is refinable
![image](https://github.com/microsoft-search/pnp-modern-search/assets/30471813/fc9e450c-5604-4ad3-b876-f86ec4fdb507)
While office number isn't, I'm not great at configuring search but to my knowledge that appears to be the problem
![image](https://github.com/microsoft-search/pnp-modern-search/assets/30471813/aae4fc8c-8695-4a8f-a954-570c69e82010)

I've added the same People:OFFICE crawled property to RefinbleString00
![image](https://github.com/microsoft-search/pnp-modern-search/assets/30471813/336dcdf7-3970-4307-91c4-12e2ac34fe3a)

And added it to the Selected Properties of the Search Results web part
![image](https://github.com/microsoft-search/pnp-modern-search/assets/30471813/4dfb9c63-d5cf-4e76-a1df-15a41053f1a1)

And then to the Filter web part
![image](https://github.com/microsoft-search/pnp-modern-search/assets/30471813/4ecdf421-bd8f-4909-bf82-aaeac2b50f9a)

Its not showed up straight away, but possibly the search engine needs time to do its stuff. I'll wait and see what happens

**Reply by AndyTuke** [reply]
Success, its popped up this afternoon. I'll mark my previous response as the answer as I've seen a few other discussions around this exact point. 

![image](https://github.com/microsoft-search/pnp-modern-search/assets/30471813/b245749a-3db1-4067-aedf-2c6812783059)

**Reply by WilliamBruno** [reply]
Hello @AndyTuke 


I am experiencing the same situation here. I'm having trouble searching for the user's location. I tried using the managed properties Office, BaseOfficeLocation, and Location but got no results. I tried following your recommendation, but I didn't quite understand the steps. Can you help me? It seems to be the same issue I'm facing.

In summary, I added the "People: Office" property to the RefinableString02 property. Is that correct? What do I need to do next? 

Thanks for your help.
